### DISCUSSION TITLE
currently always installs all languages.

### DISCUSSION THREAD

**Discussion by Aethandor** [rocksdanister/lively#924] [Ideas]
I was just browsing the installed folder looking to back-up the current install prior to updating and noticed that there are around 90 folders with language abbreviations containing a couple of identically named dll folders, presumably language specific.
If there is a language selector at the point of installation could this be used to determine which language folders+files need to be installed to save unnecessary installation of the other 90 odd languages as well ?  
I know I'm never going to need Hungarian, or Norwegian, Greek or Japanese.
It's not a major issue as the file sizes aren't massive, but it would feel 'neater'!  :)

**Comment by rocksdanister** [comment]
>I was just browsing the installed folder looking to back-up the current install prior to updating

This is not necessary, all previous versions of Lively can be found here:
https://github.com/rocksdanister/lively/releases

Wallpapers and settings are stored in separate location:
`C:\Users\<UserName>\AppData\Local\Lively Wallpaper`
and are not deleted during uninstall without permission of the user.

>noticed that there are around 90 folders with language abbreviations containing a couple of identically named dll folders, presumably language specific.
If there is a language selector at the point of installation could this be used to determine which language folders+files need to be installed to save unnecessary installation of the other 90 odd languages as well ?

Some of them are from the libraries used (modernwpf ui) and will be difficult to remove and like you said it does not take much filesize so not worth spending time on this.

**Reply by Aethandor** [reply]
thank you for your reply.  It's helpful to know that the previous versions are all available.  I did have issues with the version prior to v1.7.4.0 locking up quite regularly, so it's good to be able to revert if necessary.
I was referring to the language files in created addition to the actual in-use language in C:\Program Files (x86), but as you say, due to sizes of the files it's not a major concern. :)

**Reply by rocksdanister** [reply]
Since Lively is a compiled application which requires building, selectively removing files like that afterwards may not be a good idea.
